xml/System.Windows.Forms.Design/ComponentActionsType.xml

```diff
@@ -13,7 +13,7 @@
     <BaseTypeName>System.Enum</BaseTypeName>
   </Base>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Smart tag items that can be associated with a component.</summary>
```

Comment:
```suggestion
    <summary>Designer Action items that can be associated with a component.</summary>
```

Reply:
@merriemcgaw tiny nit - I edited the capital A in Action, made it lowercase.